docs/charter.md

```diff
@@ -10,7 +14,43 @@ ICU4X will be built from the start with several key design constraints:
 3. Availability and ease of use in multiple programming languages.
 4. Written by i18n experts to encourage best practices.
 
-ICU4X will provide an API surface in the target client-side platforms, including the web platform, iOS, Android, WearOS, WatchOS, Flutter, and Fuchsia, supported in programming languages including Rust, JavaScript, Objective-C, Java, Dart, and C++.
+# Scope
+
+ICU4X aims to provide a set of APIs sufficient to internationalize basic modern software.
```

Comment:
"aim" and "objective" seem somewhat speculative, I know this is still early, but shouldn't this be written with the 'final' product in mind?